layouts/partials/pages/about.html

```diff
@@ -46,20 +46,12 @@ <h2 class="margin-bottom-16">Monthly Meetups</h2>
 				<button class="outline">View our events</button>
 			</div>
 		</div>
-		<div class="d-flex justify-content-between margin-bottom-48 what-we-do-item">
-			<div style="background: url(https://scontent.fmnl6-1.fna.fbcdn.net/v/t1.0-9/81040016_1251664141705321_4898711282590416896_o.jpg?_nc_cat=102&_nc_ohc=5E9VoKuKursAX-2msXh&_nc_ht=scontent.fmnl6-1.fna&oh=2b688a29fe2845f70cd67abc7dd296d7&oe=5EC25375); background-size: cover; background-position: center;" class="border-radius-12 bg-secondary margin-right-32 featured-image flex-grow-1"></div>
-			<div>
-				<h2 class="margin-bottom-16">Workshops and Keynotes</h2>
-				<p class="margin-bottom-24">{{ .Site.Data.info.workshops }}</p>
-				<button class="outline">View our workshops</button>
-			</div>
-		</div>
 		<div class="d-flex justify-content-between margin-bottom-48 what-we-do-item">
 			<div style="background: url(https://scontent.fmnl6-1.fna.fbcdn.net/v/t1.0-9/81040016_1251664141705321_4898711282590416896_o.jpg?_nc_cat=102&_nc_ohc=5E9VoKuKursAX-2msXh&_nc_ht=scontent.fmnl6-1.fna&oh=2b688a29fe2845f70cd67abc7dd296d7&oe=5EC25375); background-size: cover; background-position: center;" class="border-radius-12 bg-secondary margin-right-32 featured-image flex-grow-1"></div>
 			<div>
 				<h2 class="margin-bottom-16">Design Conferences</h2>
 				<p class="margin-bottom-24">{{ .Site.Data.info.conferences }}</p>
-				<button class="outline">Visit our conference website</button>
+				<button class="outline" onclick="window.open('https://roadshow.uxph.org')">Visit UXPH Roadshow</button>
```

Comment:
Just wrap this under an `a` tag instead of using `onclick` function. Make sure to set the `target` to `blank`.